app/src/main/res/layout/fragment_profile_details.xml

```diff
@@ -0,0 +1,129 @@
+        android:layout_height="237dp"
+        android:layout_marginBottom="1dp"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_follow_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <ImageView
+            android:id="@+id/fragment_profile_details_activity_image_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:adjustViewBounds="true"
+            android:contentDescription="TODO" />
+    </HorizontalScrollView>
+
+    <TextView
+        android:id="@+id/fragment_profile_details_profile_description"
+        android:layout_width="385dp"
+        android:layout_height="87dp"
+        android:text="Ma description de moi-même"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_level"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.584" />
+
+    <TextView
+        android:id="@+id/fragment_profile_details_profile_email"
+        android:layout_width="162dp"
+        android:layout_height="40dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginBottom="62dp"
+        android:ems="10"
+        android:inputType="textEmailAddress"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_user_progress"
+        app:layout_constraintEnd_toStartOf="@+id/fragment_profile_details_profile_phone"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/fragment_profile_details_profile_description" />
+
+    <TextView
+        android:id="@+id/fragment_profile_details_profile_phone"
+        android:layout_width="206dp"
+        android:layout_height="40dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginBottom="62dp"
+        android:ems="10"
+        android:inputType="phone"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_user_progress"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/fragment_profile_details_profile_description" />
+
+    <ProgressBar
+        android:id="@+id/fragment_profile_details_user_progress"
+        style="?android:attr/progressBarStyleHorizontal"
+        android:layout_width="381dp"
+        android:layout_height="28dp"
+        app:layout_constraintBottom_toTopOf="@+id/fragment_profile_details_recent_activity_horizontal_scroll"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
+
+    <TextView
+        android:id="@+id/fragment_profile_details_level"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="6dp"
+        android:text="[user] level"
```

Comment:
As Cirrus tells us, we should preferably use the @string resource

Reply:
Yeah even better!